azureenergylabelerlib/entities.py

```diff
@@ -599,16 +582,6 @@ def is_skipped(self):
         """The finding is skipped or not."""
         return self.compliance_state.lower() == 'skipped'
 
-    @property
```

Comment:
Maybe still leave this in as we have left it on the aws one? Theodoor mentioned some future plans for this.

Reply:
As discussed, if this property is needed in the future we can add it back.